proxy/coco/src/keystore.rs

```diff
@@ -108,10 +108,10 @@ impl Keystorage<Memory> {
 }
 
 /// Synonym for an error when interacting with a file-backed store for [`librad::keys`].
-type FileError = file::Error<SecretBoxError<Infallible>, keys::IntoSecretKeyError>;
+pub type FileError = file::Error<SecretBoxError<Infallible>, keys::IntoSecretKeyError>;
```

Comment:
Why do we need to make this type public? I couldn’t find any reference to it

Reply:
The `cargo doc` command started complaining in CI: https://github.com/radicle-dev/radicle-upstream/pull/1190#pullrequestreview-524064290

Reply:
Sorry, didn’t see your comment

Reply:
No worries, concurrency is hard :smile: